test/scrabBag_spec.js

```diff
@@ -0,0 +1,66 @@
+const scrabBag = require('../public/javascripts/scrabBag')
+const expect = require('chai').expect
+
+describe('ScrabBag', function() {
+
+  it('should instantiate with @letters as object with all letters and values', function() {
+    scrabBag.clear()
+    expect(scrabBag.letters).to.be.an('object')
```

Comment:
This test is testing the *implementation* of the `ScrabbleBag` class, not the behavior.  This test is tightly coupled to the underlying data storage implementation, which means we could not use this test suite if we modified the class.  Instead, we should program to the interface that is exposed by the `ScrabbleBag` class - this way, if we change the underlying data representation used by `ScrabbleBag` the test suite (and any client code that only relies on the API) would remain unchanged.